src/Ledgers.jl

```diff
@@ -132,6 +157,18 @@ function add_account!(ledger::Ledger, acc::Account)
     ledger.indexes[id(acc)] = length(ledger.accounts)
 end
 
+function example()
```

Comment:
I think this could better be added to runtests, even as a function like this, so that one could do: include("test/runtests.jl") to get it, without making the package itself have any unnecessary code.

Reply:
Yeah. Maybe instead of runtests.jl, we can have an examples.jl and `include` that in runtests.jl.

Reply:
Yes, that's a good idea - and you can write a lot more examples, would be useful addition to the documention